crates/uv-cli/src/lib.rs

```diff
@@ -421,7 +421,7 @@ pub enum CacheCommand {
     ///
     ///
     /// By default, the cache is stored in  `$XDG_CACHE_HOME/uv` or `$HOME/.cache/uv` on Unix and
-    /// `{FOLDERID_LocalAppData}\uv\cache` on Windows.
```

Comment:
Would be you be willing to update all of the `FOLDERID_` references? I think there are some more.

You'll need to update the generated reference with `cargo dev generate-all`